lib/Models/CkanItemReference.ts

```diff
@@ -124,16 +127,16 @@ export class CkanDatasetStratum extends LoadableStratum(
     if (this.ckanItemReference.useResourceName) return this.ckanResource.name;
     if (this.ckanDataset) {
       if (
-        this.ckanItemReference.useDatasetNameAndFormatWhereMultipleResources &&
+        this.ckanItemReference.useCombinationNameWhereMultipleResources &&
         this.ckanDataset.resources.length > 1
       ) {
-        return this.ckanDataset.title + " - " + this.ckanResource.format;
+        return this.ckanDataset.title + " - " + this.ckanResource.name;
       }
       if (
-        this.ckanItemReference.useCombinationNameWhereMultipleResources &&
+        this.ckanItemReference.useDatasetNameAndFormatWhereMultipleResources &&
         this.ckanDataset.resources.length > 1
       ) {
-        return this.ckanDataset.title + " - " + this.ckanResource.name;
+        return this.ckanDataset.title + " - " + this.ckanResource.format;
```

Comment:
Switched the order that these are checked so the default is checked last. Otherwise setting `useCombinationNameWhereMultipleResources` without setting `useDatasetNameAndFormatWhereMultipleResources` to false doesn't do anything

Reply:
This would better be inline comments in ckanitemreference so it's not lost to the PR